nyc/people.py

```diff
@@ -1,142 +1,184 @@
+
+                party = web.get('Political Party')
 
-        o = Organization('Committee on Mental Health, Developmental Disability, Alcoholism, Drug Abuse and Disability Services',
-                         classification='committee',
-                         parent_id={'name' : 'New York City Council'})
-        o.add_source("http://legistar.council.nyc.gov/Departments.aspx")
+                if party == 'Democrat':
+                    party = 'Democratic'
 
-        yield o
+                if party:
+                    p.add_party(party)
 
-        o = Organization('Subcommittee on Drug Abuse',
-                         classification='committee',
-                         parent_id={'name' : 'Committee on Mental Health, Developmental Disability, Alcoholism, Drug Abuse and Disability Services'})
-        o.add_source("http://legistar.council.nyc.gov/Departments.aspx")
+                if web.get('Photo'):
+                    p.image = web['Photo']
 
-        yield o
+                contact_types = {
+                    "City Hall Office": ("address", "City Hall Office"),
+                    "City Hall Phone": ("voice", "City Hall Phone"),
+                    "Ward Office Phone": ("voice", "Ward Office Phone"),
+                    "Ward Office Address": ("address", "Ward Office Address"),
+                    "Fax": ("fax", "Fax")
+                }
 
-            
+                for contact_type, (type_, _note) in contact_types.items():
+                    if web.get(contact_type) and web(contact_type) != 'N/A':
+                        p.add_contact_detail(type=type_,
+                                             value= web[contact_type],
+                                             note=_note)
+
+                if web.get('E-mail'):
+                    p.add_contact_detail(type="email",
+                                         value=web['E-mail']['url'],
+                                         note='E-mail')
+
+                if web.get('Web site'):
+                    p.add_link(web['Web site']['url'], note='web site')
+
+                if web.get('Notes'):
+                    p.extras = {'Notes': web['Notes']}
+
+                source_urls = self.person_sources_from_office(term)
+                person_api_url, person_web_url = source_urls
+                p.add_source(person_api_url, note='api')
+                p.add_source(person_web_url, note='web')
+
+            members[member] = p
+
+        committee_types = ['Committee',
+                           'Inactive Committee',
+                           'Select Committee',
+                           'Subcommittee',
+                           'Task Force',
+                           'Land Use']  # Committee on Land Use
+
+        body_types = {k: v for k, v in self.body_types().items()
+                      if k in committee_types}
+
+        for body in self.bodies():
+            if body_types.get(body['BodyTypeName']) \
```

Comment:
Are you checkint to see if this key is in body_types or if the key exists AND the value is Truthy. If it's just the first then do a check like "BodyTypeName" in body_types.